tslint.json

```diff
@@ -8,6 +8,7 @@
     "interface-name": [true, "never-prefix"],
     "arrow-parens": false,
     "member-ordering": false,
+    "ordered-imports": false,
```

Comment:
I like of line the ordered import. It's fixable with `yarn lint --fix`. But if you prefer to remove it, it's fine.

I think it only affect ordering of "groups" and imports from one file. So you can always leave one blank line to separate groups and preserve your ordering.

Reply:
Really? I always found it a bit silly. Maybe I shouldn't, but I tend to order imports by "importance", for example `Type, TNumber, TString`, which is symmetric to `ASExpr, ASExprList, ASExprSymbol`. When ordering alphabetically, it gets pretty randomised.

Reply:
I use groups for that. Inside a group I assume the same importance.

For example:

Group 1:  compiler, reader, syntax
Group 2: debug

Although I dont mind much. I haven't used it in a large codebase yet.

 The most useful case for me seems to be importing many functions from some file/lib,  the order seems too arbitrary always.

Let's wait a bit and adjust it when we have more code.